### ISSUE TITLE
Split the metric configuration tab in two parts: one for configuration and one for targets and technical debt

### ISSUE THREAD

**Issue by fniessink** [ICTU/quality-time#4165]
# User story

As a quality manager I want the metric configuration parameters split into two groups, namely the real configuration parameters metric type, name, tags, direction, scale, unit, and targets on the one hand, and the more technical debt related parameters on the other hand so I can focus on either group depending on whether I'm configuring the metric or managing technical debt.

## Acceptance criteria

- [x] The user can switch between metric configuration parameters and technical debt parameters. When working on one set, the other set is not visible.

## Open issues

- From a work flow perspective it makes sense that the configuration tab is the leftmost one, because that is what needs to be done first. But from a frequency perspective it makes sense that the technical debt tab is the leftmost one, because it will probably be used more often than the configuration tab. Should we pick one order or change the order once the metric is configured?

## Out of scope

- Making the technical debt parameters inactive or inaccessible when the metric is informative.

# Tasks

### Frontend

- [x] Move the technical debt parameters to a "technical debt" tab.

## Documentation

- [x] Update changelog
- [x] Update to user manual
- [x] Update screenshots

# Related stories

- #1206
